config/routes.rb

```diff
@@ -3,4 +3,16 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  #guests
+  get "/guests/:id", to: "guests#show"
+
+  #guest_rooms
+  post "/guests/:guest_id/guest_rooms", to: "guest_rooms#create"
```

Comment:
YES!!!!! Nice job keeping your routes RESTful. This is exactly how I would have done it.